requirements/analyzer.txt

```diff
@@ -16,3 +16,7 @@ https://github.com/explosion/spacy-models/releases/download/en_core_web_md-3.4.0
 #######################################################################
 # This installs PyTorch which is ~250MB
 spacy-transformers==1.1.8
+
+
+sentence-transformers
+pgvector
```

Comment:
Because we import this directly in `models.py`, this probably has to go in the base requirements.

Reply:
Actually this is probably just going to be a nightmare for testing. Having a field be a postgres specific field may require our testing setup to change since our tests are run with an in-mem sqlite setup.

Reply:
FWIW, sqlite has a similar extension: https://github.com/asg017/sqlite-vss -- but might be worth just running tests in postgres? 🤷

Reply:
Looks like there isn't an easy way to use the sqlite package in Django. Another idea I had that probably makes sense:

Break the embeddings out into their own model, with a FK or OneToOne to the AnalyzedURL? That way we could keep this all self-contained.

Reply:
I think it's fine to have the embeddings on `AnalyzedUrl`. I think we just have to change the test skip logic ([this](https://github.com/readthedocs/ethical-ad-server/blob/6f77b2c783575feb5cee121853de4f3a0c87d778/adserver/analyzer/tests.py#L26-L28)) for the analyzer. We could ensure that `adserver.analyzer` is excluded from testing entirely and when it is tested that it uses Postgres. Thoughts?

Reply:
Yea, that likely makes sense as well, if we don't have tests for the code currently that we'd be skipping.